src/semidiscretization/semidiscretization_coupled.jl

```diff
@@ -494,10 +500,14 @@ function copy_to_coupled_boundary!(boundary_condition::BoundaryConditionCoupled{
         j_node = j_node_start
```

Comment:
```suggestion
        j_node = j_node_start
        element_id = linear_indices[i_cell, j_cell]
```
To shorten the code later